app/talkbot.hoon

```diff
@@ -743,14 +743,14 @@
   name
 
 ++  random  ::  Random number >=min, <max
-  |=  {min/@ max/@}
+  |=  [min=@ max=@]
   ^-  @
   =+  rng=~(. og eny)
   =^  r  rng  (rads:rng max)
   (add min r)
 
 ++  chance  ::  Has perc% chance of returning true.
-  |=  perc/@
+  |=  perc=@
   ^-  ?
   (lth (random 0 101) perc)
 
```

Comment:
Does this updated Talkbot app work? If so, you should submit a PR to @Fang-'s [Talkbot repo](https://github.com/Fang-/talkbot).

Reply:
It won't be functional even if it compiles, considering new talk. If you feel this is an important piece of newcomer-grade software to keep around in a working state, I'd be happy to monkey-patch it to work with new talk.

Reply:
Nah, doesn't matter that much. Let's just get rid of it then for now and port it later.